### ISSUE TITLE
Native API: Get InstallationName

### ISSUE THREAD

**Issue by thiagombrasil** [IQSS/dataverse#9391]
**Overview of the Feature Request**
Extend the `api/info/server` endpoint to return the Dataverse Installation name

**What kind of user is the feature intended for?**
All users roles

**What inspired the request?**
In the [Dataverse plugin for OJS](https://github.com/lepidus/dataversePlugin), it is necessary to inform the user which Dataverse installation is configured in the application.

As discussed in the [dataverse-community Google Group](https://groups.google.com/g/dataverse-community/c/XsLPpXOpaMI), the InstallationName is set to the root collection by default, so it's possible to get using the ```/api/dataverses/:root``` endpoint in most cases.

However, if the  :InstallationName setting to change the default is set, there isn’t currently an API way to get that value except for the restricted admin api: ```/api/admin/settings/:InstallationName```

**Comment by pdurbin** [comment]
@thiagombrasil good idea. The new frontend ( https://github.com/IQSS/dataverse-frontend ) is very likely going to need to know the installation name.

Also, we recently started documenting some rules for ourselves on exposing settings like this (mostly for the new frontend): https://preview.guides.gdcc.io/en/develop/developers/api-design.html#exposing-settings

<img width="879" alt="Screenshot 2023-10-08 at 11 38 32 AM" src="https://github.com/IQSS/dataverse/assets/21006/ef0ba6f6-365e-4b19-887b-5fd0990e4c48">
